sites/cnfpt/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix javascript build which was forgotten in Dockerfile and missing from image
```

Comment:
Version 0.6.0 was not broken so there is nothing to fix. The next release will be functional so everything is fine.

Reply:
Yes but I have to mention the fix in each site CHANGELOG because `bin/ci check_configuration` is configured to generate site workflow if Dockerfile change that sounds legit to check that new Dockerfile do not introduce bugs.

Reply:
Oh I see :laughing: maybe time to use --ignore-changelog?

Reply:
Yep but it did not check funcorporate CHANGELOG. I updated `bin/ci update` to support that
https://github.com/openfun/richie-site-factory/pull/146/commits/83dfbde65e5ea299a1a09fea2c228f9b3c8ee4fa